classes/Router.php

```diff
@@ -170,6 +170,23 @@ public function assertRouteIsValid(Route $route, User $player): void {
 
                 throw new RuntimeException("You must be in your village to access this page!");
             }
+
+            if ($route->village_ok === Route::VILLAGE_OR_ARENA && !$player->location->equals($player->village_location)) {
+                // hardcoding location, #dontlookplz
+                $arena_coords = new TravelCoords(16, 15, 1);
```

Comment:
hmm... Okay but let's at least hardcode this in one place like System or TravelManager and reference it from there here and in healing page